examples/complete/main.tf

```diff
@@ -0,0 +1,31 @@
+provider "azurerm" {
+  version = "~>2.3"
+  features {}
+}
+
+locals {
+  unique_name_stub = substr(module.naming.unique-seed, 0, 5)
+}
+
+module "naming" {
+  source = "git::https://github.com/Azure/terraform-azurerm-naming"
+}
+
+resource "azurerm_databricks_workspace" "test_ws" {
```

Comment:
Is there a reason we don't create the resource group here? Is it because you want the SPN to only be scoped to an existing RG or something?